forge/src/result.rs

```diff
@@ -79,33 +79,39 @@ impl TestResult {
     }
 }
 
-/// Used gas by a test
+/// Data report by a test.
 #[derive(Debug, Clone, Eq, PartialEq)]
-pub enum TestKindGas {
-    Standard(u64),
-    Fuzz { runs: usize, mean: u64, median: u64 },
+pub enum TestKindReport {
+    Standard { gas: u64 },
+    Fuzz { runs: usize, mean_gas: u64, median_gas: u64 },
+    Invariant { runs: usize, calls: usize, reverts: usize },
```

Comment:
how does the `calls` & `reverts` help me understand what happened beyond the `runs` which indicates that the program ran for some time?

Reply:
An high number of `reverts` **probably** means that you have to better constrict your search space/filter things.
If everything succeeds, the number of `calls` is equal to the user defined depth. However, if there are failures, a low number of calls **can** mean that it's an easy thing to break. But the same could be said of `runs`. So, to be honest, I'm open to remove `calls`.